python/llm/example/GPU/LLM-Finetuning/QLoRA/alpaca-qlora/README.md

```diff
@@ -175,6 +175,23 @@ bash qlora_finetune_qwen15_7b_arc_1_card.sh
 
 ##### Finetuning Baichuan2-7B examples on single Arc A770
 
+Modify `modeling_baichuan.py` in model dir. Add following 2 lines into Line 234. This change fixes [Baichuan2 type mismatch issue](https://github.com/baichuan-inc/Baichuan2/issues/291).
```

Comment:
This solution requires users to download model first and modify local model file. Maybe we need to add related description? Besides, maybe we also need to modify `--base_model` in `qlora_finetune_baichuan2_7b_arc_1_card.sh` to be local path?

Reply:
Done